tests/src/Mod/Part/App/TopoShapeExpansion.cpp

```diff
@@ -53,14 +54,14 @@ TEST_F(TopoShapeExpansionTest, makeElementCompoundOneShapeReturnsShape)
 {
     // Arrange
     auto edge = BRepBuilderAPI_MakeEdge(gp_Pnt(0.0, 0.0, 0.0), gp_Pnt(1.0, 0.0, 0.0)).Edge();
-    Part::TopoShape topoShape {edge};
-    std::vector<Part::TopoShape> shapes {topoShape};
+    TopoShape topoShape {edge};
+    std::vector<TopoShape> shapes {topoShape};
 
     // Act
-    topoShape.makeElementCompound(shapes,
-                                  "C",
-                                  Part::TopoShape::SingleShapeCompoundCreationPolicy::
-                                      returnShape /*Don't force the creation*/);
+    topoShape.makeElementCompound(
+        shapes,
+        "C",
+        TopoShape::SingleShapeCompoundCreationPolicy::returnShape /*Don't force the creation*/);
```

Comment:
The point of the policy is to remove the need for the comment, so I suggest dropping it (here and elsewhere).